### ISSUE TITLE
Best Loss Weight

### ISSUE THREAD

**Issue by charlotte12l** [davyneven/SpatialEmbeddings#8]
Hi, in the config file, the loss weight is 'w_inst':1,'w_var':1,'w_seed':10. When training in this way, I can't reproduce your results on cars due to my bad seed map. 

Did you modify the weight when training? ex, first set the weight to 1 1 10 to optimize seed map then set 1 10 1 to optimize sigma map.  

Thank you very much!

**Comment by rayhou0710** [comment]
Hi, did you test its runtime by any chance? Thanks!

**Comment by davyneven** [comment]
Hi, after training for 200 epochs on the crop images, you should finetune for another 20 epochs on the original cityscapes images. with crop size of 1024x1024. This can be done by changing the resume_path options to the last checkpoint, change n_epochs to 220, set the crop_size to 1024x1024 and change the type to 'train'. This should probably solve your seedmap issues.

**Comment by charlotte12l** [comment]
@davyneven Thanks for your reply! Now I can produce your results on car. But I can't produce your multi-class results. To train for multi-class, my seedmap is 8 channels, and when clustering, I cluster the seedmaps one by one. Did you still use the same configuration of one class(e.g. weight)? And how did you cluster when it is multi-class?
